samples/client/petstore/csharp-netcore/OpenAPIClient/src/Org.OpenAPITools/Api/FakeApi.cs

```diff
@@ -257,7 +257,7 @@ public interface IFakeApiSync : IApiAccessor
         /// <param name="enumQueryString">Query parameter enum test (string) (optional, default to -efg)</param>
         /// <param name="enumQueryInteger">Query parameter enum test (double) (optional)</param>
         /// <param name="enumQueryDouble">Query parameter enum test (double) (optional)</param>
-        /// <param name="enumFormStringArray">Form parameter enum test (string array) (optional, default to $)</param>
```

Comment:
@Januson overall it's good but looks like it breaks the default value for array of enum (string). Can you please have a look?

Reply:
@wing328 Done.